scan/templates/home/index.html

```diff
@@ -80,11 +80,17 @@ <h6 class="text-uppercase text-secondary mb-0"><i class="fas fa-project-diagram
               </div>
             </div>
           </div>
+          <div class="col-md-0 col-lg-4 pt-4 pt-lg-0 mx-auto">
+            <div class="media align-items-center mt-n2" Style="max-height: 120px;">
+              <!--<canvas style="max-height: 135px; width: 100vw; display: block;" id="myChart"></canvas>-->
+            </div>
+          </div>
         </div>
       </div>
     </div>
   </div>
 
+<div id="update_Latest">
```

Comment:
Can this be called "update-latest"? (mixing CamelCase with underline looks bad)

Reply:
yes... ill make the change.